cmd/collector/app/flags/flags.go

```diff
@@ -142,7 +143,7 @@ type HTTPOptions struct {
 	// HostPort is the host:port address that the server listens on
```

Comment:
I think we should get rid of these HTTPOptions/GRPCOptions types and use confighttp/configgrpc from OTEL. That will also take care of the TLS automatically.

See examples in #6121 and #6055

Reply:
I booked https://github.com/jaegertracing/jaeger/issues/6279

Reply:
Do i need to open other pr for #6279 , idk what booking means 😅

Reply:
so we replace HTTPOptions/GRPCOptions types with the ones from OTEL in `CollectorOptions ` struct?

Reply:
yes

Reply:
Should i open a new PR for that issue or resolve in this PR ?